config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -1155,10 +1155,11 @@ spec:
                           - topic
                         type: object
                       type:
-                        description: '`type` selects the type of exporters. The available options are `KAFKA` and `IPFIX`.'
+                        description: '`type` selects the type of exporters. The available options are `KAFKA`, `IPFIX`, and `OpenTelemetry`.'
```

Comment:
That one is not needed as OpenTelemetry has been introduced in v1beta2

You can see it here: https://github.com/netobserv/network-observability-operator/blob/main/config/crd/bases/flows.netobserv.io_flowcollectors.yaml#L5641

Reply:
yeah thats what I thought. Thats why wanted to double check. Thanks for review